src/main/java/org/AdvancedJavaEindopdracht/resource/controller/UserController.java

```diff
@@ -0,0 +1,53 @@
+package org.AdvancedJavaEindopdracht.resource.controller;
+
+import org.AdvancedJavaEindopdracht.resource.dto.UserDTO;
+import org.AdvancedJavaEindopdracht.resource.model.User;
+import org.AdvancedJavaEindopdracht.resource.service.UserService;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
+
+    private final UserService service;
+
+    public UserController(UserService service) {
+        this.service = service;
+    }
+
+    @GetMapping
+    @ResponseBody
+    public ResponseEntity<List<UserDTO>> getUsers(){
+        return ResponseEntity.ok(service.getUsers());
+    }
+
+    @GetMapping("/{id}")
+    @ResponseBody
+    public ResponseEntity<UserDTO> getUser(@PathVariable("id") final Integer id){
+        return ResponseEntity.ok(service.getUser(id));
+    }
+
+    @PostMapping(consumes = MediaType.APPLICATION_JSON_VALUE)
+    @ResponseBody
+    public ResponseEntity<UserDTO> postUser(@Valid @RequestBody User user){
+        System.out.println(user);
+        return ResponseEntity.ok(service.create(user));
+    }
+
+    @PutMapping("/{id}")
+    public ResponseEntity<Void> putUser(@PathVariable("id") final Integer id, @Valid @RequestBody User user){
+        service.update(user, id);
+        return ResponseEntity.noContent().build();
```

Comment:
Is het niet mooier om hier return ResponseEntity.ok(service.update(user, id)); te doen en dan de geupdate user returnen ipv void?